### DISCUSSION TITLE
Enhance Chia wallet security: When Chia will be available on Ledger wallet?

### DISCUSSION THREAD

**Discussion by goodbadandugly** [Chia-Network/chia-blockchain#6867] [Ideas]
After reading a lot of concerns about GUI wallet's security, I contacted the Ledger support team requesting to make Chia available on my Ledger. In their reply, the support team informed that the normal practice that the crypto's developers build apps for Ledger hardware wallets. Accordingly, I would like to request Chia's developers to get in touch with Ledger's team. Here is the relevant part of the reply from Ledger's team: 

"I understand that you want to know if CHIA is supported by Ledger devices.
I am sorry to inform you that this crypto asset is currently not supported. Ledger hardware wallets benefit from a secure operating system called BOLOS, providing an open platform upon which developers can build applications. Developers willing to build apps for Ledger hardware wallets, like adding support for the crypto asset you're referring to, are welcome to get in touch with us.
Please check our list of supported crypto assets to see which assets are supported on what Ledger device: http://ledger.com/cryptocurrencies
A public project overview is available here: https://trello.com/b/5nQ1mdzt/ledger-roadmap
Thank you for reaching out to Ledger Support."

Hope other members of Chia community could support such hardware security enhancement for Chia.

**Comment by loppefaaret** [comment]
We are already on the road for Ledger support, it is just not as important as say pooling protocol.
it is on our trello board:
https://trello.com/b/ZuNx7sET/engineering-core
under these 2 entries:
https://trello.com/c/SFqSU52s/854-confirm-any-requirements-coming-out-of-the-ledger-nano-s-x-project
https://trello.com/c/GWhZwZ3N/749-add-support-for-ledger-wallet

**Reply by KralMurat** [reply]
> We are already on the road for Ledger support, it is just not as important as say pooling protocol.
> it is on our trello board:
> https://trello.com/b/ZuNx7sET/engineering-core
> under these 2 entries:
> https://trello.com/c/SFqSU52s/854-confirm-any-requirements-coming-out-of-the-ledger-nano-s-x-project
> https://trello.com/c/GWhZwZ3N/749-add-support-for-ledger-wallet

Any progress on this?

**Comment by goodbadandugly** [comment]
Thanks a lot for answering! Pool is certainly important but, having 1100 plots, I don't think I would ever join any pool that would imply replotting, hence I appreciate much more all your efforts in enhancing Chia security. Best!

**Comment by DatDraggy** [comment]
Has anything changed so far? The last activity on trello is now a year ago. Just wondering

**Reply by loppefaaret** [reply]
afaik, not directly - but it would not really be something that is coming from us, in form of announcement, it would be up to ledger to take that step - but surely, if they do so, we would likely follow up with one as well i'd imagine. As far as i can say, we have a line of communication going, but developing a hardware solution for BLS requires some work!
https://trello.com/c/GWhZwZ3N/749-add-support-for-ledger-wallet

**Reply by DatDraggy** [reply]
Thanks for the reply. Very well. Guess we'll just have to wait for now but I'm definitely looking forward to a ledger integration ^^
Love y'all work

**Comment by goodbadandugly** [comment]
Is ledger support something that would renew the interest to Chia? I think recent trend moving from proof of work to staking is rather global now and proof of space and time is not so attractive in crypto world. Does Chia offer anything else than proof of space and time? Look what is happening with Cosmos network with its SCRT secret bridge, staking and LPing, or with cosmos based osmosis pooling and other IBC. What about comparing Chia with the advances of Polkadot with its interoperability and layer 0 solution? What Chia may offer to make it more attractive than ETH 2.0? 35.365 EiB - this has not been moving at all for really sometime now. I think developers must something more robust than join to the useless Cryptocurrency Open Patent Alliance

**Reply by loppefaaret** [reply]
i doubt we or the community is hatching our fate to ledger support to be honest... i doubt _that_ is what will make or break us... dont get me wrong, i would absolutely love a hardware wallet to support us, but the implementation of BLS takes time.

**Reply by KralMurat** [reply]
I think you underestimate hard wallets, of course BTC ETH or any other coin will survive without Ledger, so no coin's "fate" is related with Ledger, but, it's a matter of supporting another, well known and widely accepted security protocol. 

Ledger team says "Developers should develop their app for BOLOS to be able to support Chia on Ledger" and you say "it would not really be something that is coming from us, in form of announcement, it would be up to ledger to take that step" so Ledger expects from Chia, and Chia devs expect something from Ledger. And it's waiting for more than a year. 

In Ledger's github you can see hundreds of entries asking for Chia support, here also many people asking for Chia support, there are even many shitty low profile coins already took their places in Ledger platform, but Chia still not, for years.

**Comment by Elbrain** [comment]
Any updates on this?

**Reply by HVNC1** [reply]
?

**Comment by HVNC1** [comment]
?

**Comment by mic3ael** [comment]
Any updates?

**Comment by Phoenix603** [comment]
Any updates?

**Comment by cyber-eternal** [comment]
Hi @loppefaaret. Could you please update the community regarding the integration of Chia into the Ledger wallet? I guess a lot of people wait for that. 
Thanks in advance.

**Reply by WhitePaper01** [reply]
Definitely!

**Comment by KralMurat** [comment]
Any progress on this topic?

**Comment by leokryp** [comment]
I hope Chia network never partners with Ledger or any other crap trashx in this market.

chia wallet meets all security requirements. was born ready and safe!

If you don't know how to protect yourself from yourself, stop crying!

**Reply by KralMurat** [reply]
I hope chia gets rid of this kind of dinosaur thoughts. If you call hard wallets ad trasx, you know nothing John Snow. All crypto currencies has their own security measures, chia is not the first, wont be the last, but, if you don't know the importance of hard wallet, you know nothing.

**Reply by leokryp** [reply]
if you eat garbage and want to be healthy, I really understand your side of the importance of a hospital.

**Reply by KralMurat** [reply]
If you cant afford a hard wallet, and trying to call it shitty, I understand your side as well

**Reply by fnoquiq** [reply]
I would like to know why the hard wallet is safe/unsafe?

**Comment by MumfMeisterT** [comment]
We are actively working on this. We will provide more clarity on timelines when we know more.

**Reply by KralMurat** [reply]
@MumfMeisterT thanks for the answer, at least someone from the Chia crew sees/cares. 

But the first topic is created in Chia trello in 2020, and now 2022 is almost finished. Chia has a huge community, most decentralized coin with hundreds of dedicated supporters, so if you're waiting anything from Ledger side, you can use that power as a leverage. But when we contact with Ledger, they say they have an OS and coin side (Chia) should develop a native app to be supported by Ledger. 

Developing that native app for Ledger OS (Bolos I guess) is taking so much time, even years? Or there's a technological limitation like blockchain structure is not compatible with OS or something? 

If you can provide little bit more information about what has been done in 2 years, what is blocking the process, which part is responsible, if Ledger side, what we can do to push them, that would be nice, so it'll be clear for everyone. Thanks in advance.

**Reply by KralMurat** [reply]
> We are actively working on this. We will provide more clarity on timelines when we know more.

How actively are you working on this? Any update after 4 months?

**Comment by eliliam** [comment]
Is there a place where we can track progress on ledger support? It appears the original Trello tickets have been archived, and searching for ledger in the current Trello board turns up no results.

**Comment by KralMurat** [comment]
How about at least supporting it with 3rd party support? For example, RVN and HNT does that, Ledger Live doesn't support RVN, but RVN still have a Ledger Support, so you can keep your assets in Ledger, but manage it in another wallet app, just like Helium. Helium has a Ledger App, but Ledger Live doesn't support Helium, so in Helium Wallet app, you can add Ledger account, and still keep assets in Ledger, but manage it outside of Ledger, with 3rd party app.

So Chia may have a Ledger app, Ledger users can install it, and Chia native app can add Ledger wallet inside, and, we can manage Ledger account via Chia app. This should be much easier, and not even need Ledger side to do anything or approve anything. Do you consider this option?

**Reply by Maladon0815** [reply]
I wish 100% ledger support (ledger live) too!

**Comment by ApfelBirneKreis** [comment]
Could the status be added back to Chia's trello board? For the Community to track that? I feel like the ledger support would be a huge improvement for the community

**Reply by iZakaroN** [reply]
For me the hardware wallet support is the major feature that define if crypto project has a future, and my main concern whether to stick with chia or not

**Reply by KralMurat** [reply]
Unfortunately noone knows what is the status of this or if there's any progress, every a couple of months, someone comes here and tells "we're working on it, we'll let you know when we know more details"

Since 2020, when it's first mentioned in Chia Trello, for 3 years, noone knows what has been done and what has been left. Just hearing "we're working on it"........

**Reply by iZakaroN** [reply]
The most reliable information I have is that Chia is waiting for "ledger G2 and the BLS standard support", but no more details about that.

**Reply by KralMurat** [reply]
Yes, I have the same information, from 2021.

**Comment by goodbadandugly** [comment]
I started this thread and got disappointed with chia developers. They are not serious. Chia does not have any functionality, waste of time and resources to mine chia… it will never replace Bitcoin and never have functionality as ETH. Sent from my iPhoneOn 17 Feb 2023, at 17:55, Tayfun Murat Ipek ***@***.***> wrote:﻿
Yes, I have the same information, from 2021.

**Comment by KralMurat** [comment]
Any progress on this topic, beside "we're working on this and we'll let you know soon"?

**Reply by ApfelBirneKreis** [reply]
Ledger finally implemented BLS protocol on ledger Nano X. The ball is now on Chia’s side again.

**Comment by goodbadandugly** [comment]
This is a useless token, it will never match bitcoin and developers don’t give a shit any more….Sent from my iPhoneOn 22 Mar 2023, at 16:12, Tayfun Murat Ipek ***@***.***> wrote:﻿
Any progress on this topic, beside "we're working on this and we'll let you know soon"?

**Reply by ApfelBirneKreis** [reply]
The fact that you are talking about chia being a token shows me you have no idea about the Chia Blockchain.

**Comment by tarantula3** [comment]
Useless token that will never be on Ledger

**Reply by ApfelBirneKreis** [reply]
The fact that you are talking about chia being a token shows me you have no idea about the Chia Blockchain.
